src/client/api_util.cpp

```diff
@@ -1847,9 +1847,10 @@ AttributeInfoEx &AttributeInfoEx::operator=(const AttributeConfig_5 *att_5)
 		else
 			memorized = MEMORIZED_WRITE_INIT;
 	}
+	enum_labels.clear();
 	for (unsigned int j=0; j<att_5->enum_labels.length(); j++)
 	{
-		enum_labels[j] = att_5->enum_labels[j];
+		enum_labels.push_back(att_5->enum_labels[j].in());
```

Comment:
Good idea but enum_labels is a vector<string> but att_5->enum_labels is a DevVarStringArray, not a vector<string>.